tests/README.md

```
@@ -0,0 +1,514 @@
+
+```bash
+# Show all output (including passing tests)
+bats --print-output-on-failure tests/
+
+# Tap format (for CI/CD)
+bats --formatter tap tests/
+```
+
+## Test Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)